cmd/server/main.go

```diff
@@ -134,7 +134,7 @@ func startServer(logger *log.Logger, cfg *config.Config, mux *http.ServeMux, thu
 		TLSConfig: tlsConfig,
 	}
 
-	logger.Info("Starting Asgardeo Thunder...", log.String("address", serverAddr))
+	logger.Info("Asgardeo Thunder started...", log.String("address", serverAddr))
```

Comment:
Server may not be started at this point

Reply:
Improved with https://github.com/asgardeo/thunder/compare/18e958c4fef32e306e250c33f53bc073aa9f2326..eb3fac7ab874ba7ae9b0aa2f62d0d3c931b27537